examples/clamped_truss/own_weight_clamped_truss.jl

```diff
@@ -0,0 +1,117 @@
+# ---------------------------------
+# Own weight clamped truss example
+# --------------------------------
+#=
+This model consist of a clamped truss with its own weight. First the
+weight is applied and consecutive the load at the time starting from the own
+weight deformed configuration.
+
+# Analysis 1: Own weight
+
+## Forces
+-> g
+--------> F
+
+# Analysis 1: Own weight
+#-----------------------------
+# ->  ->  ->  ->  ->  ->  -> -
+#-----------------------------
+
+# Analysis 2: Own weight + load
+#-----------------------------
+# ->  ->  ->  ->  ->  ->  -> - -----> F
+#-----------------------------
+=#
+using ONSAS, Test
+using Dictionaries: dictionary
+
+"Return problem parameters"
+function parameters()
+    N = 10     # Number of elements.
+    E = 30e6    # Young's modulus.
+    ν = 0.3     # Poisson's ratio.
+    ρ = 750.0   # Density.
```

Comment:
just noting that in the future we can also plug in physical units via  https://github.com/PainterQubits/Unitful.jl/   
an advantage is that automatically checks for consistency of units in defining equations

Reply:
Nais, this checks every mathematical operation and the outcome units ? This may be expensive in computational terms ?

Reply:
it has no runtime overhead actually > https://news.ycombinator.com/item?id=22151009